packages/devtools/lib/src/ui/analytics_platform.dart

```diff
@@ -61,6 +61,16 @@ void computeDevToolsCustomGTagsData() {
   }
 }
 
+// Look at the query parameters '&ide=' and record in GA.
+void computeDevToolsQueryParams() {
+  ga.ideLaunched = ga.ideLaunchedCLI; // Default is Command Line launch.
+  final searchParams = html.UrlSearchParams(html.window.location.search);
```

Comment:
use
`dart
    final Uri uri = Uri.parse(window.location.toString());
    uri.queryParameters[ga.ideLaunchedQuery];
  ` 
to make the code a little closer to working with flutter:web

Reply:
Done.